affiliates/base/templates/base/home.html

```diff
@@ -17,6 +17,30 @@ <h2 class="site-logo">
 {% endblock %}
 
 {% block content %}
+
+  {% if LANG in settings.FRIENDS_MSG_LOCALES %}
```

Comment:
Also with it only showing if it's translated, localizers on Pontoon won't be able to see the message and translate it themselves since it won't show up. If you still go with this approach then I'd make sure it always shows if `settings.DEV == True`.